examples/evaluate_single_robot/main.py

```diff
@@ -15,6 +20,16 @@ def main() -> None:
     # Set up logging.
     setup_logging()
 
+    parser = argparse.ArgumentParser()
```

Comment:
Here I totally agree that this argument makes sense, but I would like to keep this example extremely minimal, as it is the first example the students see. We can instead add a constant SIMULATION_TIME? That would be nice for the students to have and I assume you use it for the tests